modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,320 @@
+
+The following sections cover the steps to deploy regular or import Sync Gateway nodes.
+// or a single replicator node.
+
+You will have to repeat the steps in this section once for the "regular" nodes and once for the "import" node. The difference is the  Sync Gateway configuration file and the Kubernetes Deployment Controller config file.
+
+=== Configuration file
+
+In this section, you will create the Sync Gateway configuration file and store it in a secret location.
+
+. Open the configuration file corresponding to your deployment.
++
+|===
+| Regular Node | Importer Node
+
+| link:{attachmentsdir}/kubernetes/sgw-config.json[sgw-config.json]
+| link:{attachmentsdir}/kubernetes/sgw-config-import.json[sgw-config-import.json]
+|===
+. Replace the `server` key with the addressable Couchbase Server hostname.
+The following example shows the configuration file for a regular node.
++
+[source,json]
+----
+{
+  "logging": {
+    "console": {
+      "log_file_path": "/var/tmp/sglogs",
+      "console": {
+        "log_level": "debug",
+        "log_keys": ["*"]
+      },
+      "error": {
+        "enabled": true,
+        "rotation": {
+          "max_size": 20,
+          "max_age": 180
+        }
+      },
+      "warn": {
+        "enabled": true,
+        "rotation": {
+          "max_size": 20,
+          "max_age": 90
+        }
+      },
+      "info": {
+        "enabled": false
+      },
+      "debug": {
+        "enabled": false
+      }
+    }
+  },
+  "databases": {
+    "db": {
+      "server": "http://cb-example-0000.cb-example:8091", // <1>
+      "bucket": "default",
+      "username": "admin", // <2>
+      "password": "password",
+      "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } },
+      "allow_conflicts": false,
+      "revs_limit": 20
+    }
+  }
+}
+----
+<1> The server key should point to the Couchbase Server cluster.
+This would typically be of the form `http://CB_SERVER_POD.CB_SERVER_SERVICE_NAME:8091`.
+<2> The username/password keys should match what was setup in the <<pre-requisites, pre-requisites>> section when you configured the Sync Gateway RBAC user.
+. You will use a https://kubernetes.io/docs/concepts/configuration/secret/[Kubernetes Secret] to pass the configuration file to Sync Gateway on launch.
+Alternatively, you could use a https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/[Kubernetes configMap] if you are not concerned about the security.
+However, since the Sync Gateway contains sensitive information, it is recommended that you create a secret from the config file and pass that to the Sync Gateway.
+Run the following command to create a secret called "sgw-config" or "sgw-config-import" with the corresponding Sync Gateway configuration file.
++
+[source,console]
+----
+kubectl create secret generic [sgw-config|sgw-config-import] --from-file [configuration-file]
+----
++
+If successful, you will see the following.
```

Comment:
@jamiltz  Again, to be consistent with command, can we have a tab that shows the output. Realize this isn't a big deal but want to keep it as straightforward as possible

Reply:
https://github.com/couchbase/docs-sync-gateway/commit/a230695a63497984aad1f70dcec4f393d55f0804